packages/hooks/src/use-clipboard.ts

```diff
@@ -7,29 +7,37 @@ import copy from "copy-to-clipboard"
  * @param text the text or value to copy
  * @param timeout delay (in ms) to switch back to initial state once copied.
  */
-export function useClipboard(text: string, timeout = 1500) {
+export function useClipboard(
+  text: string,
+  timeout:
+    | number
+    | { timeout?: number; format?: "text/plain" | "text/html" } = 1500,
```

Comment:
Shouldn't we allow any mimetype (any string) for `format`, so users could supply `application/json` for example?

Reply:
you are right, seems like we can support multiple `mime/types` https://www.w3.org/TR/clipboard-apis/#mandatory-data-types-x 

Should we pass any string? or create an ENUM?

Reply:
I would allow any `string`, same what `copy-to-clipboard` does.